### DISCUSSION TITLE
useId modules error

### DISCUSSION THREAD

**Discussion by Fiyuva** [nuxt/nuxt#28835] [Questions]
Hello, I get this error when I try to build after the latest update.

ERROR  Nuxt Build Error: node_modules/nuxt/dist/app/composables/id.js (1:9): "useId" is not exported by "node_modules/vue/dist/vue.runtime.esm-bundler.js", imported by "node_modules/nuxt/dist/app/composables/id.js".

in dev mode it says

Uncaught (in promise) SyntaxError: The requested module '/_nuxt/node_modules/vue/dist/vue.runtime.esm-bundler.js?v=66f24f9d' does not provide an export named 'useId' (at id.js:1:10)

`{
  "type": "module",
  "private": true,
  "scripts": {
    "build": "nuxt build",
    "dev": "nuxt dev",
    "generate": "nuxt generate",
    "preview": "nuxt preview",
    "postinstall": "nuxt prepare"
  },
  "devDependencies": {
    "@fortawesome/fontawesome-svg-core": "^6.6.0",
    "@fortawesome/free-brands-svg-icons": "^6.6.0",
    "@fortawesome/free-regular-svg-icons": "^6.6.0",
    "@fortawesome/free-solid-svg-icons": "^6.6.0",
    "@fortawesome/vue-fontawesome": "^3.0.8",
    "@nuxtjs/color-mode": "^3.4.4",
    "@nuxtjs/i18n": "^8.5.1",
    "@types/bootstrap": "^5.2.10",
    "@vite-pwa/nuxt": "^0.10.4",
    "bootstrap": "^5.3.3",
    "eslint": "^9.9.1",
    "eslint-plugin-security-node": "^1.1.4",
    "nuxt": "^3.13.1",
    "nuxt-aos": "^1.2.5",
    "sweetalert2": "^11.6.13",
    "typescript": "^5.5.4",
    "vue-tsc": "^2.1.4"
  },
  "dependencies": {
    "@pinia/nuxt": "^0.5.4",
    "@types/crypto-js": "^4.2.2",
    "crypto-js": "^4.2.0",
    "eslint-plugin-anti-trojan-source": "^1.1.1",
    "flag-icons": "^7.2.3",
    "nuxt-purgecss": "^2.0.0",
    "pinia": "^2.2.2",
    "sass": "^1.78.0",
    "vue-recaptcha": "^3.0.0-alpha.6"
  },
  "resolutions": {
    "@nuxt/kit": "3.0.0-rc.13"
  },
  "overrides": {
    "vue": "latest"
  }
}
`

**Comment by Twitch0125** [comment]
Try adding `"vue": "latest"` to your dependencies or devDependencies. Including that only in `overrides` wasn't working for me, but adding it to devDepenencies did.

**Comment by Fiyuva** [comment]
This solution worked for me:

Remove package_lock and folder node_modules
Reinstall your dependencies

(Works leaving with overrides vue :) )
